senior-knights/course-schedulizer
# Fix semester length radio buttons

To Test:
- Upload the math schedule CSV, go to the interim, and click on a class.
- The update section popup should show that the section is an intensive, and it should show the intensive letter A. (Previously there was a bug which prevented the intensive letter radio buttons from displaying).
- Now try updating the section to be a different semester length (like a second half class or an intensive C class). When you change the `Semester Length` from `Intensive` to `Half`, `First` should be set by default (rather than nothing being selected).
- Click on that section again to display the update section popup again. Your change should have persisted. (There was a bug previously which caused the changes to not persist). Try exporting as well to ensure that the change is reflected in the CSV.

Closes #75
